maquetado/styles.css

```diff
@@ -0,0 +1,27 @@
+body{
+  margin: 0;
+}
+
+.seccion-principal{
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  padding-top:50px;
+  border-top:8px solid;
+  border-color: #f5c85f;
```

Comment:
you can set the color as the last param in the previous property like this: `border-top: 8px solid #F5C85F;`